api/app/settings/common.py

```diff
@@ -617,6 +617,8 @@
 
 ENABLE_DB_LOGGING = env.bool("DJANGO_ENABLE_DB_LOGGING", default=False)
 if ENABLE_DB_LOGGING:
+    if not DEBUG:  # pragma: no cover
+        raise ImproperlyConfigured("DB logging will not work without DEBUG=True")
```

Comment:
why not set `DEBUG` to `True` if `DJANGO_ENABLE_DB_LOGGING` is set instead?

Reply:
Yeah, I did consider that, and in hindsight I think that's probably better